crd/client/gcpfirewall/clientset/versioned/scheme/register.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 The Kubernetes Authors.
+Copyright 2022 The Kubernetes Authors.
```

Comment:
Copyrights should be kept at the year the file was originally introduced. Is there something else going on here I'm missing?

Reply:
Copyrights are generated by code-gen as we made some changes to types.go.  Do we want to manually change it back to 2021? Or this has been address by the latest code-gen?

Reply:
Oh interesting. I see that generated files in kuberneteskubernetes ([example](https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/abac/zz_generated.deepcopy.go)) don't have copyright dates at all. But I don't know offhand how to get the same treatment in this repo.  Would you open an issue about this?  It's fine for this PR to update the dates for now and we'll figure it out separately.